docs/theming.md

```diff
@@ -382,7 +402,7 @@ In addition, we also define custom properties for known dark and light backgroun
 
 
 Ideally, we should set all of the text colors on primary, since we never know which one an MDC-Web component might use.
-Since we're just using buttons, though, let's keep it simple for now:
+So let's keep it simple for now:
```

Comment:
Suggestion: "Since our cards only contain text and no components, let's keep it simple for now:"

Reply:
DONE